hsm/m0hsm_api.c

```diff
@@ -68,11 +68,11 @@ static struct m0_realm  *m0_uber_realm;
 			fprintf(options.log_stream, _fmt, ##__VA_ARGS__)
 #define VERB(_fmt, ...) if (options.trace_level >= LOG_VERB) \
 			fprintf(options.log_stream, _fmt, ##__VA_ARGS__)
-#define DEBUG(_fmt, ...) if (options.trace_level >= LOG_DEBUG) \
```

Comment:
Andriy, 
can we move all  thease  #define  ERROR, INFO, VERB, DBG to
hsm/m0hsm_api.h

Reply:
Please don't move it there. ERROR, INFO can be quite commonly used macros therefore let's not overflow global namespace.